code/datums/diseases/xeno_transformation.dm

```diff
@@ -36,18 +37,32 @@
 			if (prob(10))
 				to_chat(affected_mob, pick(SPAN_DANGER("Your skin feels very tight."), SPAN_DANGER("Your blood boils!")))
 				affected_mob.take_limb_damage(3)
+				playsound(affected_mob.loc, 'sound/voice/alien_growl1.ogg', 15, FALSE)
 			if (prob(5))
-				affected_mob.whisper(pick("Soon we will be one...", "Must... devour...", "Capture...", "We are perfect.", "Hsssshhhhh!"))
+				if(!ismonkey(affected_mob))
+					affected_mob.whisper(pick("Soon we will be one...", "Must... devour...", "Capture...", "We are perfect.", "Hsssshhhhh!"))
 			if (prob(8))
 				to_chat(affected_mob, SPAN_DANGER("You can feel... something...inside you."))
 		if(5)
 			to_chat(affected_mob, SPAN_DANGER("Your skin feels impossibly calloused..."))
 			affected_mob.apply_damage(10, TOX)
 			affected_mob.updatehealth()
 			if(prob(40))
-				var/turf/T = find_loc(affected_mob)
-				gibs(T)
-				var/mob/living/carbon/human/H = affected_mob
-				H.Alienize(XENO_T1_CASTES)
-				src.cure()
+				var/turf/turf = get_turf(affected_mob)
+				var/area/area = get_area(turf)
+				gibs(turf)
+				for(var/obj/item/items in affected_mob)
+					affected_mob.drop_inv_item_on_ground(items)
+				var/mob/living/carbon/xenomorph/drone/xeno = new()
+				xeno.forceMove(turf)
+				xeno.set_hive_and_update(XENO_HIVE_FERAL)
+				if(ismonkey(affected_mob))
+					xeno.free_for_ghosts(TRUE)
+				else
+					affected_mob.mind.transfer_to(xeno, TRUE)
```

Comment:
```suggestion
					affected_mob.mind?.transfer_to(xeno, TRUE)
```

Reply:
may be actually unneeded but doesnt seem like so

Reply:
not sure this really matters but sure